src/opm/parser/eclipse/EclipseState/Runspec.cpp

```diff
@@ -328,6 +329,11 @@ Runspec::Runspec( const Deck& deck ) :
                 OpmLog::note(msg);
             }
         }
+        if (runspecSection.hasKeyword("CO2STOR")) {
```

Comment:
Cool if you use the compiled constants - that particularly makes sense for the more volatile opm specific keywords:
```C++
#include <opm/parser/eclipse/ParserParserKeywords/C.hpp>

....
if (runspsecSection.hasKeyword<ParserKeywords::CO2STOR>()) {
    ...
    ...
}
```